client.py

```diff
@@ -0,0 +1,131 @@
+import requests
```

Comment:
This PR only adds the function for gateway APIs(all tested). We still need few extra PRs to add docs, sample cases, and handle the errors of failed call situations.